server/libs/domain/src/asset/asset.service.ts

```diff
@@ -0,0 +1,18 @@
+import { AssetType } from '@app/infra/db/entities';
```

Comment:
the `server/apps/immich/src/api-v1/asset/asset.service.ts` will eventually be refactored in here, correct?

Reply:
Yes. Basically everything left depends on asset service and that's too much to move all at once, so we will have both for a short time.